presto-session-property-managers/src/main/java/io/prestosql/plugin/session/db/util/SessionPropertiesDaoUtil.java

```diff
@@ -0,0 +1,31 @@
+/*
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package io.prestosql.plugin.session.db.util;
+
+public final class SessionPropertiesDaoUtil
+{
+    private SessionPropertiesDaoUtil() {}
+
+    public static final String SESSION_SPECS_TABLE = "session_specs";
```

Comment:
Let's inline these, like we do everywhere else. It makes the queries easier to read.